12.LOCO_BULK_MLP/0.data_splits/scripts/0.LOCO_data_split.py

```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
```

Comment:
Consider a markdown cell at the start of the file with a title + info about this notebook and what it is doing.

Reply:
addressed